Automattic/wp-calypso
# Plugins: Display eligibility warnings for dotcom sites

**Update**: it seems that after the rebase https://github.com/Automattic/wp-calypso/pull/9880 is making it impossible to get to the warnings page from plugin detail page. If you want to test the component you can navigate directly to: `plugins/bbpress/{siteSlug}/eligibility`

**Testing instructions**:
1. Dispatch this action from your console:

```javascript
dispatch( { 
    type: actionTypes.AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST,
    siteId: '{siteId}',
} );
```

2. navigate to `plugins/bbpress/{siteSlug}/eligibility` (due to bug mentioned above, getting here from by clicking on `Install` button wont work atm)
3. Verify that correct errors and warnings are shown for you test site.




**Visual changes:**
* Plugin detail page for dotcom site:

![step1](https://cloud.githubusercontent.com/assets/1182160/21201170/34060c92-c24a-11e6-99b4-6f5679a76992.png)

* Eligibility warnings component:

![step2](https://cloud.githubusercontent.com/assets/1182160/21201174/3995fc6c-c24a-11e6-82fe-70cffd4f8d28.png)
